hack/jenkins/e2e.sh

```diff
@@ -190,6 +204,194 @@ case ${JOB_NAME} in
     MINION_SIZE="n1-standard-1"
     NUM_MINIONS="2"
     ;;
+
+  # GCE master+node upgrade step 1: deploy a cluster at latest release.
+  kubernetes-upgrade-gce-step1-deploy)
+    : ${E2E_CLUSTER_NAME:="gce-upgrade"}
```

Comment:
nit: maybe alphabetize the variable names in each of these lists. (it's what I did for the gce e2e jobs, at least)